draft-pauly-add-requirements.md

```diff
@@ -186,68 +186,82 @@ Using a content-provider's encrypted resolver can also provide several privacy a
 to communicate with a provisioned unencrypted DNS resolver and discover the equivalent or associated encrypted
 DNS resolver.
 
-For the second group, mechanisms additionally need to provide some information about the applicability and capabilities
-of encrypted resolvers. This information could either be provisioned,
+Domain-specific resolver discovery mechanisms additionally need to provide some information about the
+applicability and capabilities of encrypted resolvers. This information could either be provisioned,
 or be discovered based on clients actively trying to access content.
 
 # Privacy and security requirements {#priv-sec}
 
-Since one of the main reasons for enabling encrypted DNS on clients is to improve the privacy and security properties
-of DNS queries, discovery mechanisms must be designed carefully to not weaken a client's security or privacy.
+A primary goal of encrypted DNS is improving the privacy and security of DNS queries and answers in the presence
+of malicious attackers. Such attackers are assumed to interfere with or otherwise impede DNS traffic and corresponding
+discovery mechanisms. They may be on-path or off-path between the client and entities with which the client
+communicates. These attackers can inject, tamper, or otherwise interfere with traffic as needed.
+Given these capabilities, an attacker may have a variety of goals, including, though not limited to:
 
-- Clients are always responsible for selecting which resolver(s) to use. Standards describing resolver discovery
-mechanisms must not place any requirements on clients to select some resolvers instead of other resolvers.
+- Monitor and profile clients by observing unencrypted DNS traffic
 
-- User or administrator preference must always be able to override any discovery mechanism. Networks
-may block connectivity to the resolver chosen by the user, but that does not mean that systems should ignore user
-preference.
+- Modify unencrypted DNS traffic to filter or augment the user experience
 
-- Automatic discovery mechanisms must be tied to an entity that is already otherwise involved in providing
-service to the client, such as: a network provider, a VPN server, a content provider being accessed, or
-a server that the client has manually configured.
+- Block encrypted DNS
 
-- The relationship between a discovered resolver and the client-known entity must be securely authenticated.
+Clients cannot assume that their network does not have such an attacker unless given some means of authenticating or otherwise
+trusting the communication with their DNS resolver.
 
-- Discovery mechanisms must not allow an attacker not associated with the client-known entity to successfully
-redirect DNS traffic to themselves.
+Given this type of attacker, resolver discovery mechanisms must be designed carefully to not worsen a client's security or
+privacy posture. In particular, attackers must not be able to:
```

Comment:
are we MUSTing here (and below)?

Reply:
This is informational, I don't want normative language

Reply:
Works for me, but should we consider rewording or just leave it? Either is fine